refinery/ui/source/js/dashboard/partials/collaboration-card.html

```diff
@@ -68,7 +68,7 @@ <h3>
            aria-labelledby="headingOne"
            data-parent="#accordion">
             <ul class="list-unstyled dashboard-member-list">
-              <li ng-repeat="member in group.member_list">
+              <li ng-repeat="member in group.member_list | orderBy: last_name">
```

Comment:
Typically I like the API to handle sorting (it's faster) but decided to it here until we decide what order we'd like. Last_name seems like a good default.

Reply:
Never mind, after talking to Scott, it's really a client side filter.